atomic_reactor/utils/resource_manager.py

```diff
@@ -0,0 +1,132 @@
+"""
+Copyright (c) 2022 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import os
+import subprocess
+import logging
+import json
+from datetime import datetime
+from shlex import quote
+
+ROOT_PATH = 'resource_manager/'
+logger = logging.getLogger(__name__)
+
+
+class ResourceManager:
+    '''
+        Manages build resources
+    '''
+
+    def __init__(self):
+        pass
+
+    def check_avail(self, user, host):
+        try:
+            data = os.popen(f'ssh {user}@{host} -4C "cat {ROOT_PATH}info.json"').read()
```

Comment:
Please use os.path.join() for ROOT_PATH and info.json

Reply:
Do we want to use popen here or some python library like paramiko?

Reply:
+1 for paramiko or equivalent

Reply:
paramiko (or equiv.) also sidesteps security issues with popen/shell

Reply:
we agreed to use popen in first iteration, improvements later (but before release)

Reply:
OK, I see. Either way, please do something like

```python
    args = ['ssh', f'{user}@{host}', '-4C', f'"cat {ROOT_PATH}info.json"']
    subprocess.check_output(args, shell=False)
```

instead

(This is off the top of my head, and probably won't work as intended, but should convey the idea)

Reply:
In case we don't use paramiko or any equivalent at this moment, I believe we at least need `-o StrictHostKeyChecking=no` to establish the ssh connection.